yarn-remote-client/yarn_is_ready.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
This file doesn't belong in this directory, yarn-remote-client, which is for configuration files to be used by YARN remote clients.
I suggest to move it to the root folder and add the following line in Dockerfile:
`COPY yarn_is_ready.sh /etc/yarn_is_ready.sh`